### PR TITLE
Added MMLanScan library

### PR DESCRIPTION
**Pull request by mavris**
## Project URL
https://github.com/mavris/MMLanScan

## Description
Added MMLanScan library

## Checklist

- [ x] Only one project/change is in this pull request
- [ x] Addition in chronological order (bottom of category)
- [ x] Supports iOS 7 or later
- [ x] Has a commit from less than 2 years ago
- [ x] Has a **clear** README in English

### CODE REVIEW

**Comment by danger-awesome-ios** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>

    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="true">Found 1 link issue</td>
    </tr>
  </tbody>
</table>

#### Link issue by [`awesome_bot`](https://github.com/dkhamsing/awesome_bot)

  Line | Status | Link
| ---: | :----: | --- |
825 | [301](https://httpstatuses.com/301) | https://www.urbanairship.com/products/mobile-engagement/engage redirects to<br>http://www.urbanairship.com/products/mobile-app-engagement


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

**Comment by mavris** [comment]
Hi,
I don't get the error. If you look the commit you will notice that I just added a library

**Comment by dkhamsing** [comment]
Link issue is unrelated

File: README.md [hunk 1]
```diff
@@ -804,7 +804,7 @@ Also see [push notifications](#push-notifications)
 * [Malibu](https://github.com/hyperoslo/Malibu) - Networking library built on promises :large_orange_diamond:
 * [YTKNetwork](https://github.com/yuantiku/YTKNetwork) - YTKNetwork is a high level request util based on AFNetworking.
 * [UnboxedAlamofire](https://github.com/serejahh/UnboxedAlamofire) - Alamofire + Unbox: the easiest way to download and decode JSON into swift objects. :large_orange_diamond:
-
+* [MMLanScan](https://github.com/mavris/MMLanScan) - An iOS LAN Network Scanner library
```
Comment (dkhamsing) [inline README.md]: Can you add a line break after your line
Thanks

**Pull request closed without merging**